main/antivirus/stubs/freshclam.profile.mas

```diff
@@ -2,6 +2,7 @@
   $observerScript
 </%args>
 /run/clamav/clamd.ctl rw,
+/proc/*/status r,
```

Comment:
Could you add this change to the ChangeLog? It is relevant for security purposes.